docs/docs/end-users/VisualStudio.md

```diff
@@ -4,14 +4,26 @@ categoryindex: 1
 index: 7
 ---
 # Visual Studio
-TODO: Explain version of fantomas used in Visual Studio.
-The F# Formatting extension sets up Fantomas as the default formatter for F# files, configurable from Visual Studio's options.
+The F# Formatting extension sets up Fantomas as the default formatter for F# files, configurable from Visual Studio's options.  
+Do note that the extension might not be up to date with the latest version of Fantomas.  
 
 * [Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=asti.fantomas-vs)
 * [Visual Studio 2022](https://marketplace.visualstudio.com/items?itemName=asti.fantomas-vs22)
 
+Aditionally, you can set Fantomas as an external tool: 
```

Comment:
I think this is more the guidance for VsForMac instead of an alternative for Visual Studio.
The image is also from VsForMac, so maybe more all the under the `### Visual Studio for Mac` section.